packages/shared-ux/button_toolbar/src/popover/popover.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('<ToolbarPopover />', () => {
     expect(button.prop('color')).toBe('text');
     expect(button.prop('css')).toMatchObject({
       backgroundColor: '#FFF',
-      border: '1px solid #D3DAE6 !important',
```

Comment:
Since all styles are localised to the component through `EUI` and our style modification applied through the `css` prop gets merged with the declaration that ships with the EUI component. With no possibility of accepting further style modifications or running into specificity issues, we don't need to set `!important`.